crates/stackable-webhook/src/servers/conversion.rs

```diff
@@ -165,6 +175,7 @@ impl ConversionWebhookServer {
         crds_and_handlers: impl IntoIterator<Item = (CustomResourceDefinition, H)>,
         options: ConversionWebhookOptions,
         client: Client,
+        maintain_crds: bool,
```

Comment:
note: I feel like this should be part of the `ConversionWebhookOptions`.

Reply:
As the `field_manager` is also on the `ConversionWebhookOptions`, I think I actually agree with @Techassi.
Went with his suggestion in 72404a4